src/lein_monolith/task/fingerprint.clj

```diff
@@ -288,8 +288,9 @@
                               [[k subproject]]
                               [k
                                (update subproject :profiles merge
-                                       (plugin/build-inherited-profiles
-                                         monolith subproject))]))
+                                       (into {}
+                                             (plugin/build-profiles
+                                               monolith subproject)))]))
                           subprojects)]
```

Comment:
Open question: should we care about profile ordering for fingerprinting? I'm biasing toward keeping the existing behavior of not guaranteeing the order is deterministic for now but I'm open to addressing this here while I'm at it

Reply:
Seems like it's probably okay to not have ordering matter here, we're just adding the information so the fingerprinting code can see all of the relevant information by including profiles.